code/game/machinery/bots/secbot.dm

```diff
@@ -26,10 +26,11 @@
 	radio_frequency = SEC_FREQ //Security channel
 	bot_type = SEC_BOT
 	model = "Securitron"
+	var/movement_delay = 4
```

Comment:
This was originally a magic number further down the code. I'm thinking of refactoring all the bots so they share the same movement variables